### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Gunnu14**
changes requested

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -10,3 +11,4 @@
 
 </body>
 </html>
+beautiful code
```
Comment (nitibilkhu) [inline index.js:16]: Rewrite to dull code

**Review by nitibilkhu — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (Gunnu14) [reply index.js:16]: ok

File: main.css [hunk 2]
```diff
@@ -1,5 +1,8 @@
 <!DOCTYPE html>
 <html>
+
+border style should be attractive
```
Comment (nitibilkhu) [inline main.css:4]: Add more content please

**Comment by Gunnu14** [comment]
ok

**Pull request merged**